.github/workflows/test.yml

```
@@ -45,6 +45,7 @@
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./coverage.txt
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)